public/jsx/views/complete-payment.jsx

```diff
@@ -18,6 +19,10 @@ module.exports = React.createClass({
     userEmail: React.PropTypes.string.isRequired,
   },
 
+  componentDidMount: function() {
+    tracking.setPage('/payment/complete');
```

Comment:
does this have to be a URL? Could we do ``tracking.setPage('CompletePayment')`` ?